src/components/SearchResultList/SearchResultList.tsx

```diff
@@ -45,10 +45,10 @@ const SearchResultList = () => {
     setSearchKeyWord(event.target.value);
   };
 
-  const listSearchResults = async (queryParams: PaginationParams, keyword: string) => {
+  const listSearchResults = async (paginationParams: PaginationParams, keyword: string) => {
     setLoading(true);
     try {
-      const data = await listSearchResultsAction(queryParams, { keyword });
+      const data = await listSearchResultsAction(paginationParams, { keyword });
```

Comment:
We missed that variable naming but usually avoid using `data`:

```suggestion
      const fetchedSearchResult = await listSearchResultsAction(paginationParams, { keyword });
```